java8/src/main/java/org/jetbrains/annotations/BlockingContext.java

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.annotations;
+
+import java.lang.annotation.*;
+
+/**
+ * Indicates that the annotated execution context (e.g. coroutine dispatcher, scheduler, etc.) allows blocking calls inside.
```

Comment:
I'm not very happy with Javadoc. Say, I'm a Java user, know nothing about Kotlin. Is it useful for me? In any case, a code example would be nice. Please also add `@see` tags to link these annotations to each other.

Reply:
For java users it is useless yet (until reactor support is implemented) I'll add an example for kotlin